packages/Main/src/Core/PointCloudNode.js

```diff
@@ -16,6 +16,15 @@ class PointCloudNode extends THREE.EventDispatcher {
         this.sse = -1;
     }
 
+    get pointSpacing() {
+        return this.layer.spacing / 2 ** this.depth;
+    }
+
+    get id() {
+        // hierarchyKey for potreeNode
+        return this.hierarchyKey ?? `${this.depth}${this.x}${this.y}${this.z}`;
```

Comment:
This returns something only if in a `PotreeNode` and `Potree2Node`. This should not be defined in the more generic `PointCloudNode`.